docs/modules/picard.md

```diff
@@ -13,54 +13,24 @@ sequencing data.
 
 Supported commands:
 
-* `MarkDuplicates`
-* `InsertSizeMetrics`
+* `AlignmentSummaryMetrics`
```

Comment:
I reordered these to be alphabetical and also fixed the order of sections below this to match the order in this list

Reply:
Great! Except `R` comes after `Q` in the alphabet usually 😅